src/main/resources/db/changelog/db.changelog.xml

```diff
@@ -538,9 +538,17 @@
         </createTable>
     </changeSet>
 
-    <changeSet id="045-add-moderation-required-column-to-blog-table" author="vlosrutere">
+    <changeSet id="046-add-moderation-required-column-to-blog-table" author="vlosrutere">
         <addColumn tableName="BLOG">
             <column name="MODERATION_REQUIRED" type="boolean"/>
         </addColumn>
     </changeSet>
+
+    <changeSet id="047-update-moderation-required-column-in-blog-table" author="vlosrutere">
```

Comment:
I think that SQL commands in `047` and `048` should be a part of `046` changeset as they all form a one, logical change in the database.